community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -204,14 +204,18 @@
                   "statistics used to create the plan have changed more than this value, " +
                   "the plan will be considered stale and will be replanned. Change is calculated as " +
                   "abs(a-b)/max(a,b). This means that a value of 0.75 requires the database to approximately " +
-                  "quadruple in size. A value of 0 means always replan, and 1 means never replan." )
+                  "quadruple in size. A value of 0 means replan as soon as possible, with the soonest being " +
+                  "defined by the cypher.min_replan_interval which defaults to 10s. After this interval the " +
+                  "divergence threshold will slowly start to decline, reaching 10% after about 7h, so that " +
+                  "long running databases will still get query replanning on even modest changes, while not " +
+                  "replanning frequently unless the changes are very large." )
```

Comment:
This sentence is a bit on the long side, and is packed with lots of dense information to boot. I would suggest seeing if it can be split into two sentences (whilst retaining the useful info it currently conveys)